Source/Core/Core/HW/DSPHLE/UCodes/AX.cpp

```diff
@@ -18,17 +18,21 @@
 AXUCode::AXUCode(DSPHLE* dsphle, u32 crc) : UCodeInterface(dsphle, crc), m_cmdlist_size(0)
 {
   INFO_LOG(DSPHLE, "Instantiating AXUCode: crc=%08x", crc);
-  m_mail_handler.PushMail(DSP_INIT);
-  DSP::GenerateDSPInterruptFromDSPEmu(DSP::INT_DSP);
-
-  LoadResamplingCoefficients();
 }
 
 AXUCode::~AXUCode()
 {
   m_mail_handler.Clear();
 }
 
+void AXUCode::Initialize()
+{
+  m_mail_handler.PushMail(DSP_INIT);
```

Comment:
Can you maybe try replacing these two lines with PushMail(DSP_INIT, true)?

This has smarter interrupt scheduling behavior in the case of messages being queued.